Xamarin.Forms.Platform.UAP/TitleViewManager.cs

```diff
@@ -52,6 +52,17 @@ void commandBarUnloaded(object sender, RoutedEventArgs e)
 			{
 				CommandBar.LayoutUpdated -= commandLayoutUpdated;
 				CommandBar.Unloaded -= commandBarUnloaded;
+				CommandBar.Loaded += CommandBar_Loaded;
+			}
+		}
+
+		private void CommandBar_Loaded(object sender, RoutedEventArgs e)
```

Comment:
Also please drop `private`

Reply:
Hallo,

for a sample do following:

1. Create a new Xamarin-Forms-Project from the default VS2019 template
2. Just select UWP and use a MasterDetails page as start steup.
3. Open the ItemsPage.xaml.cs file and add following three lines in the ItemsPage constructor after the InitializeComponent() call:

			ContentView titleView = new ContentView();
			titleView.BackgroundColor = Color.Red;
			NavigationPage.SetTitleView(this, titleView);

this defines a simple red title view which should always resize with the main window width.
4. Start the project:
![image](https://user-images.githubusercontent.com/2929116/83752845-7a308b00-a669-11ea-838e-e8294693f64e.png)
5. Click the "Add" command toolbar item -> a modal page is shown:
![image](https://user-images.githubusercontent.com/2929116/83752920-a3511b80-a669-11ea-8aca-7551c888f5a0.png)

6. Click the "Cancel" command toolbar item -> the preivious page is shown again
7. Now after a resize of the main window the custom title view width is not updated any more:
![image](https://user-images.githubusercontent.com/2929116/83753027-d3002380-a669-11ea-9279-a5622981edd2.png)

Reply:
Hello,

do you have seen the sample? Is there any progress on this request? Many thanks for your further help.

Regards,
Thomas.